apps/custom-config/continuousDialog.js

```diff
@@ -0,0 +1,47 @@
+'use strict'
+var property = require('@yoda/property')
+var BaseConfig = require('baseConfig')
+
+var SWITCH_OPEN = 'open'
+var SWITCH_CLOSE = 'close'
+
+var PICKUP_SWITCH_OPEN = '当前不支持连续对话'
+var PICKUP_SWITCH_CLOSE = '当前不支持连续对话'
+var CONFIG_FAILED = '设置失败'
+
+class ContinuousDialog extends BaseConfig {
+  getIntentMap () {
+    return {
+      pickupswitch: this.applyPickupSwitch.bind(this)
+    }
+  }
+
+  getUrlMap () {
+    return {
+      continuousDialog: this.onPickupSwitchStatusChanged.bind(this)
+    }
+  }
```

Comment:
A tip: if you don't mind provide the way `dialog.intentMap` or `dialog.urlMap`, you could use the getter as the following:

```js
class ContinuousDialog extends BaseConfig {
  get intentMap () {
    return ...
  }
}
```